manifests/w/WinFsp/WinFsp/1.9/WinFsp.WinFsp.yaml

```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: WinFsp.WinFsp
+PackageVersion: 1.9

```

Comment:
```suggestion
PackageVersion: 1.9.21096
```

Reply:
Was trying to follow previous listings versioning scheme, as it dropped the numbers after the second digit